qiskit/algorithms/gradients/param_shift_estimator_gradient.py

```diff
@@ -43,7 +41,24 @@ def __init__(self, estimator: BaseEstimator, options: Options | None = None):
                 default options > primitive's default setting.
                 Higher priority setting overrides lower priority setting
         """
-        self._gradient_circuits = {}
+        self._parameter_value_offset_cache = {}
+        self._SUPPORTED_GATES = [
```

Comment:
This could be defined on class level as it's common to each instance, e.g.
```python
class ...:
  SUPPORTED_GATES = [ ...]
  
  def __init__(...)
```
or if it stays here we should probably choose lowercased variable name 🙂